Examples/JS/flow/ImageType.js

```diff
@@ -0,0 +1,19 @@
+//
+//  ImageType.js
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+//  @flow
```

Comment:
Does this work? I think Flow only looks at the first 10 tokens in a file by default. (Might want to move `// @flow` to the top just in case)